### ISSUE TITLE
Share this track : No apps can perform this action

### ISSUE THREAD

**Issue by dwids** [rcgroot/open-gpstracker#447]
Hi. Not sure if this app is still being maintained, but I can no longer Share any of my tracks.  I have been using this app for years on assorted phones but it stopped letting me do this recently.  I used it ok - to Share - about a month ago on the same phone and tried again today. It didn't work.

App V 1.5.4
Pixel XL with Android 7.1.2 (Security patch level 7.1.2)
Error: Share this track: No apps can perform this action

I do have a number of tracks I would like to 'upload' (share). As I don't have Root access, I can't see any way of Sharing them etc. and am hoping you have ideas. Thanks.

**Comment by johnjohndoe** [comment]
Did you try the [Open GPS Tracker - Exporter](https://play.google.com/store/apps/developer?id=Ren%C3%A9+de+Groot)?

**Comment by dwids** [comment]
No, I didn't know it existed. But thank you very much for the quick and helpful response. It worked first time and all my tracks have been exported safely to Google Drive. I really appreciate your help as I was a bit worried!

**Comment by johnjohndoe** [comment]
You are welcome. I have been through this myself. @rcgroot could maybe advertise the exporter app in the main app to the user.

**Comment by n1mh** [comment]
Hi,

what if I don't want to export the GPX files to google drive? I want to be able to export them to the mobile phone storage.

**Comment by rcgroot** [comment]
@n1mh you could try the alpha [Open GPS Tracker - NG](https://play.google.com/store/apps/details?id=nl.sogeti.android.gpstracker.ng) it uses the tracks stored in your current OGT and has simple track share feature.

**Comment by n1mh** [comment]
@rcgroot yes, that's what I've finally done and it exports right. I just wanted to highlight that some of us don't use google drive. In my case, I use the app in an old phone and export the gpx to a compute so, that option won't work for me.

Regards.
